common/src/main/java/icu/nullptr/hidemyapplist/common/app_presets/SuspiciousAppsPreset.kt

```diff
@@ -23,13 +23,34 @@ class SuspiciousAppsPreset : BasePreset(NAME) {
         "com.happymod.apk",
         "com.pd.pdhelper",
         "cm.aptoide.pt",
+        "com.offsec.nethunter.store",
```

Comment:
You shouldn't add apps into multiple lists (you added it in "Rooted Apps" already)